apb-test.sh

```diff
@@ -0,0 +1,272 @@
+    $CMD run "$pod_name" \
+        --namespace=$apb_name \
+        --env="POD_NAME=$pod_name" \
+        --env="POD_NAMESPACE=$apb_name" \
+        --image=$apb_image \
+        --image-pull-policy=Never \
+        --restart=Never \
+        --attach=true \
+        --serviceaccount=$apb_name \
+        -- $action --extra-vars '{ "namespace": "'$apb_name'", "cluster": "'$CLUSTER'" }'
+    printf "\n"
+    $CMD get all -n $apb_name
+    echo -en 'travis_fold:end:'$pod_name'\\r'
+    printf "\n"
+}
+
+function setup_openshift() {
+    printf ${green}"Testing APB in OpenShift"${neutral}"\n"
+    echo -en 'travis_fold:start:openshift\\r'
+    printf ${yellow}"Setting up docker for insecure registry"${neutral}"\n"
+    sudo apt-get update -qq
+    sudo cat /etc/docker/daemon.json
+
+    #{"registry-mirrors": ["https://mirror.gcr.io"], "mtu": 1460}
+    # sudo echo "{ \"insecure-registries\": [\"172.30.0.0/16\"] }" >  /etc/docker/daemon.json
+    # sudo rm -rf  /etc/docker/daemon.json
+    sudo sed -i "s/\DOCKER_OPTS=\"/DOCKER_OPTS=\"--insecure-registry=172.30.0.0\/16 /g" /etc/default/docker
+    sudo cat /etc/default/docker
+    sudo iptables -F
+    sudo service docker restart
+    sudo ps faux | grep docker
+
+    printf "\n"
+
+    printf ${yellow}"Bringing up an openshift cluster and logging in"${neutral}"\n"
+#    sudo docker cp $(docker create docker.io/openshift/origin:$OPENSHIFT_VERSION):/bin/oc /usr/local/bin/oc
+#    sudo cp /usr/local/bin/oc /usr/local/bin/kubectl
+    if [ "$OPENSHIFT_VERSION" == "v3.11" ]; then
+        wget https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
+        tar -xzvf openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz --strip-components=1 openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit/oc
+
+    elif [ "${OPENSHIFT_VERSION}" == "v3.10" ]; then
+        wget https://github.com/openshift/origin/releases/download/v3.10.0/openshift-origin-client-tools-v3.10.0-dd10d17-linux-64bit.tar.gz
+        tar -xzvf openshift-origin-client-tools-v3.10.0-dd10d17-linux-64bit.tar.gz --strip-components=1 openshift-origin-client-tools-v3.10.0-dd10d17-linux-64bit/oc
+
+    elif [ "${OPENSHIFT_VERSION}" == "v3.9" ]; then
+        wget https://github.com/openshift/origin/releases/download/v3.9.0/openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz
+        tar -xzvf openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz --strip-components=1 openshift-origin-client-tools-v3.9.0-191fece-linux-64bit/oc
+    fi
+    sudo cp oc /usr/local/bin/
+    sudo cp /usr/local/bin/oc /usr/local/bin/kubectl
+    oc cluster up \
+        --skip-registry-check=true \
+        --routing-suffix=172.17.0.1.nip.io \
+        --public-hostname=172.17.0.1 \
+        --enable=service-catalog,template-service-broker,router,registry,web-console,persistent-volumes,sample-templates,rhel-imagestreams
```

Comment:
Why do you need the web-console or sample-templates?

Reply:
it is not called, I just keept it as on the original script.
the lines which call those functions are commented.